base/console/src/main/java/com/netscape/admin/certsrv/Console.java

```diff
@@ -1762,7 +1778,35 @@ public static void mainImpl(String argv[]) throws Exception {
         */
         }
 
-        UtilConsoleGlobals.initJSS();
+	//cfu
+	CryptoManager manager = null;
+	/*
+	if (nssdb_path == null)
+            UtilConsoleGlobals.initJSS();
+	else {
+            try {
+                CryptoManager.initialize(nssdb_path);
+            } catch (AlreadyInitializedException e) {
+                // it is ok if it is already initialized
+            } catch (Exception e) {
+                log("INITIALIZATION ERROR: " + e.toString());
+                System.exit(1);
+            }
+        }
+	*/
+	if (nssdb_path != null) {
```

Comment:
With the suggested code above the `nssdb_path` will never be `null`, so this check is not necessary.